Xamarin.Forms.Platform.WinRT/CellControl.cs

```diff
@@ -249,15 +249,17 @@ void SetCell(object newContext)
 				}
 				else
 				{
-					string textContent = newContext.ToString();
-
+					IItemsView<Cell> controller = lv;
 					if (isGroupHeader)
 					{
 						TemplatedItemsList<ItemsView<Cell>, Cell> group = GetGroup(newContext, lv);
-						textContent = GetDisplayTextFromGroup(lv, group);
+						cell = controller.CreateDefault(GetDisplayTextFromGroup(lv, group));
+					}
+					else
+					{
+						cell = controller.CreateDefault(newContext);
+						cell.BindingContext = newContext;
```

Comment:
This is the same logic as above on line [248](https://github.com/xamarin/Xamarin.Forms/pull/197/files#diff-95bf3f15932f690e96bcbdfacda3ff9dR248).